src/new/wallet.js

```diff
@@ -100,6 +101,8 @@ class HathorWallet extends EventEmitter {
 
     this.onConnectionChangedState = this.onConnectionChangedState.bind(this);
     this.handleWebsocketMsg = this.handleWebsocketMsg.bind(this);
+
+    this.firstConnection = true;
```

Comment:
Add comments explaining what it is used for.

Reply:
Done!